aptos-move/framework/aptos-framework/sources/chain_id.move

```diff
@@ -21,4 +21,18 @@ module aptos_framework::chain_id {
     public fun get(): u8 acquires ChainId {
         borrow_global<ChainId>(@aptos_framework).id
     }
+
+    #[test_only]
```

Comment:
Why would you ever need to change the chain id in move unit tests? May be you're looking for initialize_for_test so you can initialize the chain id once?

Reply:
I don't have to be able to change it, but I would like to specify.
I agree with you, "initialize_for_test" is enough. I fixed. 2560a3e
At first I thought of this flow.
```txt
genesis::setup();
chain_id::test_get();
```